developer/src/kmc-model/package.json

```diff
@@ -30,7 +30,6 @@
   "dependencies": {
     "@keymanapp/keyman-version": "*",
     "@keymanapp/models-types": "*",
-    "typescript": "^4.9.5",
     "xml2js": "^0.4.19"
   },
```

Comment:
Typescript needs to be a `dependency` not a `devDependency` because we use typescript to compile models.